### PR TITLE
Ignore path params in context switcher label

### PR DESCRIPTION
**Pull request by parostatkiem**
**Description**

Changes proposed in this pull request:

- Ignore path params in context switcher label
- Unit tests of the new feature

**Related issue(s)**
Fixes https://github.com/kyma-project/luigi/issues/335

### CODE REVIEW

File: core/src/navigation/services/context-switcher.js [hunk 1]
```diff
@@ -62,7 +63,11 @@ export const ContextSwitcherHelpers = {
     }
 
     // we are inside the context switcher base path
-    const selectedId = currentPath.replace(parentNodePath, '').split('/')[0];
+    const selectedId = currentPath
+      .replace(parentNodePath, '')
+      .split('/')[0]
+      .split(nodeParamPrefix)[0] //ignore everything after nodeParamPrefix
```
Comment (pekura) [inline core/src/navigation/services/context-switcher.js]: I would remove .split(nodeParamPrefix)[0] and just let the .split('?')[0] there. There is no rule that forbids nodeParamPrefix to be a character not allowed in a path segment.

Comment (parostatkiem) [reply core/src/navigation/services/context-switcher.js]: Would you like to see this?
image

Comment (pekura) [reply core/src/navigation/services/context-switcher.js]: yes!

Comment (parostatkiem) [reply core/src/navigation/services/context-switcher.js]: Allright then...

**Pull request merged**